### PR TITLE
Logic flow

### PR DESCRIPTION
**Pull request by Esterni**
Changes made are focused on navigating the package and how to use it. Minor modifications to the responses from logs for authentication. Mostly added docstrings to explain what functions do.

### CODE REVIEW

File: pyracing/client.py [hunk 1]
```diff
@@ -42,37 +42,71 @@ def default_logger():
...
     # Wrapper for all functions that builds the final self.session.get()
+
     async def build_request(self, url, params, retry=True):
+        """ Builds the final GET request from url and params
+        """
+
+        # TODO Log the URL with query string instead of params dict.
         self.log.info('Making get request to url: ' +
-                      url + ' with params: %s', params)
+                      url + f' with params: {params}')
```
Comment (XanderRiga) [inline pyracing/client.py:94]: is the `f` here a typo?

Comment (XanderRiga) [reply pyracing/client.py:94]: Nevermind, just looked it up. That is pretty cool

File: pyracing/client.py [hunk 2]
```diff
@@ -82,28 +116,46 @@ async def build_request(self, url, params, retry=True):
...
+        registered drivers.
+        """
         url = ct.URL_ACTIVEOP_COUNT
         payload = {
-            'custid': custID,
+            'custid': custID,  # Purpose of custID unknown
             'maxcount': maxCount,
-            'include_empty': 'n',  # Flag is y/n
-            'excludeLite': 0
+            'include_empty': include_empty,
```
Comment (XanderRiga) [inline pyracing/client.py:133]: is the field on this url actually `include_empty`? Usually iRacing uses camelcase so I would expect it to be `includeEmpty`, but they do tend to change it up so just want to make sure.

Comment (Esterni) [reply pyracing/client.py:133]: Haha yup...😂 that's how they have it. Just 1 more tally for inconsistencies

**Pull request merged**